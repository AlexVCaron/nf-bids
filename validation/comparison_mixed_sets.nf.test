nextflow_workflow {

    name "Plugin Comparison: Mixed Sets"
    script "./main.nf"
    workflow "main_workflow"

    test("qmri_mpm - Plugin output") {
        when {
            params {
                bids_dir = "/home/local/USHERBROOKE/vala2004/dev/bids2nf/tests/data/bids-examples/qmri_mpm"
                libbids_sh = "/home/local/USHERBROOKE/vala2004/dev/bids2nf/libBIDS.sh/libBIDS.sh"
                config = "/home/local/USHERBROOKE/vala2004/dev/bids2nf/plugins/nf-bids/validation/configs/config_mpm.yaml"
            }
        }
        then {
            assert workflow.success
            assert workflow.out.bids_channel
            def results = workflow.out.bids_channel.collect()
            def json = groovy.json.JsonOutput.toJson(results)
            def serialized = new groovy.json.JsonSlurper().parseText(json)
            assert snapshot(
                groovy.json.JsonOutput.toJson(serialized)
            ).match()
        }
    }
}
