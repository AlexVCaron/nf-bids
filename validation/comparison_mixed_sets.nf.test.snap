{
    "qmri_mpm - Plugin output": {
        "content": [
            "[[[\"sub-01\",\"NA\",\"NA\",\"NA\"],{\"data\":{\"RB1COR\":{\"bodyMTw\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyMTw_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyMTw_RB1COR.nii\"},\"bodyPDw\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyPDw_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyPDw_RB1COR.nii\"},\"bodyT1w\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyT1w_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyT1w_RB1COR.nii\"},\"headMTw\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headMTw_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headMTw_RB1COR.nii\"},\"headPDw\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headPDw_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headPDw_RB1COR.nii\"},\"headT1w\":{\"json\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headT1w_RB1COR.json\",\"nii\":\"qmri_mpm/sub-01/fmap/sub-01_acq-headT1w_RB1COR.nii\"}},\"TB1EPI\":{\"json\":[[\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-01_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-02_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-03_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-04_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-05_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-06_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-07_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-08_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-09_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-10_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-11_TB1EPI.json\"],[\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-01_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-02_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-03_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-04_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-05_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-06_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-07_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-08_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-09_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-10_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-11_TB1EPI.json\"]],\"nii\":[[\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-01_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-02_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-03_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-04_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-05_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-06_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-07_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-08_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-09_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-10_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-11_TB1EPI.nii\"],[\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-01_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-02_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-03_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-04_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-05_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-06_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-07_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-08_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-09_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-10_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-11_TB1EPI.nii\"]]},\"MPM\":{\"MTw\":{\"json\":[\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-1_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-2_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-3_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-4_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-5_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-6_flip-1_mt-on_MPM.json\"],\"nii\":[\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-1_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-2_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-3_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-4_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-5_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-6_flip-1_mt-on_MPM.nii\"]},\"PDw\":{\"json\":[\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-1_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-2_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-3_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-4_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-5_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-6_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-7_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-8_flip-1_mt-off_MPM.json\"],\"nii\":[\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-1_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-2_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-3_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-4_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-5_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-6_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-7_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-8_flip-1_mt-off_MPM.nii\"]},\"T1w\":{\"json\":[\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-1_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-2_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-3_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-4_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-5_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-6_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-7_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-8_flip-2_mt-off_MPM.json\"],\"nii\":[\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-1_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-2_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-3_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-4_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-5_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-6_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-7_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-8_flip-2_mt-off_MPM.nii\"]}}},\"filePaths\":[\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyMTw_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyMTw_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyPDw_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyPDw_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyT1w_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-bodyT1w_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headMTw_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headMTw_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headPDw_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headPDw_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headT1w_RB1COR.json\",\"qmri_mpm/sub-01/fmap/sub-01_acq-headT1w_RB1COR.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-01_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-01_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-02_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-02_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-03_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-03_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-04_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-04_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-05_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-05_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-06_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-06_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-07_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-07_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-08_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-08_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-09_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-09_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-10_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-10_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-11_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-11_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-01_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-01_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-02_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-02_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-03_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-03_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-04_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-04_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-05_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-05_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-06_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-06_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-07_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-07_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-08_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-08_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-09_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-09_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-10_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-10_TB1EPI.nii\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-11_TB1EPI.json\",\"qmri_mpm/sub-01/fmap/sub-01_echo-2_flip-11_TB1EPI.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-1_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-1_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-2_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-2_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-3_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-3_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-4_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-4_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-5_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-5_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-6_flip-1_mt-on_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-MTw_echo-6_flip-1_mt-on_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-1_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-1_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-2_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-2_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-3_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-3_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-4_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-4_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-5_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-5_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-6_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-6_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-7_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-7_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-8_flip-1_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-PDw_echo-8_flip-1_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-1_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-1_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-2_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-2_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-3_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-3_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-4_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-4_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-5_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-5_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-6_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-6_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-7_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-7_flip-2_mt-off_MPM.nii\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-8_flip-2_mt-off_MPM.json\",\"qmri_mpm/sub-01/anat/sub-01_acq-T1w_echo-8_flip-2_mt-off_MPM.nii\"],\"bidsParentDir\":\"/home/local/USHERBROOKE/vala2004/dev/bids2nf/tests/data/bids-examples\",\"subject\":\"sub-01\",\"session\":\"NA\",\"run\":\"NA\",\"task\":\"NA\"}]]"
        ],
        "meta": {
            "nf-test": "0.9.3",
            "nextflow": "25.10.0"
        },
        "timestamp": "2025-10-29T11:07:02.462747768"
    }
}